quick-protobuf/tests/rust_protobuf/common/test_oneof.rs

```diff
@@ -1,8 +1,17 @@
 use quick_protobuf::*;
 
-use std::borrow::Cow;
-use rust_protobuf::hex::{encode_hex, decode_hex};
 use super::test_oneof_pb::*;
+use rust_protobuf::hex::{decode_hex, encode_hex};
+use std::borrow::Cow;
+
+fn t<F>(f: F)
```

Comment:
Any reason to move it?

Reply:
I think I pulled it out of the method so I could use it in two different files since the behavior for v2 vs v3 is slightly different.

https://github.com/tafia/quick-protobuf/pull/129/files/99d1ebe96ff2eda118788b63454fea078ccd2d0f#diff-652110135f7c9e6e9bae3be74ac02869R5
https://github.com/tafia/quick-protobuf/pull/129/files/99d1ebe96ff2eda118788b63454fea078ccd2d0f#diff-693fe99aab3c7b2174edc30026c68f6fR6